MCprep_addon/materials/prep.py

```diff
@@ -489,7 +489,12 @@ def execute(self, context):
 			self.report({'ERROR'}, "No materials found on selected objects")
 			return {'CANCELLED'}
 		_ = generate.detect_form(mat_list)
-		invalid_uv, affected_objs = uv_tools.detect_invalid_uvs_from_objs(obj_list)
+		
+		# If we have a commonmcobj header, we don't need 
+		# to check for scaled UVs
```

Comment:
Actually, is that right? shouldn't it be more that we know the answer, not that it's automatically false? (I didn't live test the before / after)

Reply:
Ah perfect, it happens downstream - then allllll goood 👍